File: process_config.dart
// Random Dart function generated on 2025-08-27

class DeleteRequest {
  final int id;
  final String name;
  final double value;

  const DeleteRequest({
    required this.id,
    required this.name,
    required this.value,
  });
}

List<DeleteRequest> delete_response(List<DeleteRequest> items) {
  const double multiplier = 6.0;
  
  return items
      .map((item) => DeleteRequest(
            id: item.id,
            name: '${item.name}_processed',
            value: item.value * multiplier,
          ))
      .toList();
}

void main() {
  final testData = [
    DeleteRequest(id: 1, name: 'Item1', value: 19.0),
  ];

  final result = delete_response(testData);
  print('Processed ${result.length} items');
}

File: process_config.cs
// Random C# function generated on 2025-08-29
using System;
using System.Collections.Generic;
using System.Linq;

public class HandleValue
{
    public int Id { get; set; }
    public string Name { get; set; }
    public double Value { get; set; }
    public DateTime CreatedAt { get; set; }
    
    public HandleValue(int id, string name, double value)
    {
        Id = id;
        Name = name;
        Value = value;
        CreatedAt = DateTime.Now;
    }
}

public static class Update_userProcessor
{
    public static List<HandleValue> Update_user(IEnumerable<HandleValue> items)
    {
        const double multiplier = 5.0;
        
        return items
            .Where(item => item.Value > 0)
            .Select(item => new HandleValue(
                item.Id,
                $"{item.Name}_processed",
                item.Value * multiplier))
            .ToList();
    }
    
    public static void Main()
    {
        var testData = new List<HandleValue>
        {
            new HandleValue(1, "Item1", 87),
            new HandleValue(2, "Item2", 68)
        };
        
        var result = Update_user(testData);
        Console.WriteLine($"Processed {result.Count} items");
    }
}
